### PR TITLE
fix s2t output log and dump result to file

### PR DESCRIPTION
**Pull request by rmittal-github**

### CODE REVIEW

File: riva/clients/nmt/riva_streaming_speech_translate_client.cc [hunk 1]
```diff
@@ -71,6 +71,7 @@ DEFINE_bool(
     "True returns text exactly as it was said with no normalization.  False applies text inverse "
     "normalization");
 DEFINE_string(ssl_cert, "", "Path to SSL client certificatates file");
+DEFINE_string(nmt_text_file, "s2t_output.txt", "NMT text file");
```
Comment (virajkarandikar) [inline riva/clients/nmt/riva_streaming_speech_translate_client.cc]: can you clarify in the help string that this is the output file and what will be the expected contents of this file?

Comment (virajkarandikar) [reply riva/clients/nmt/riva_streaming_speech_translate_client.cc]: do we always want to dump to file ?

Comment (rmittal-github) [reply riva/clients/nmt/riva_streaming_speech_translate_client.cc]: yes let me update the help string. Yes, output file is expected to be generated for each client command run.

Comment (rmittal-github) [reply riva/clients/nmt/riva_streaming_speech_translate_client.cc]: S2S client already dumps the audio output file for each run, so thinking to have the same behavior for S2T client also.

Comment (rmittal-github) [reply riva/clients/nmt/riva_streaming_speech_translate_client.cc]: updated description

File: riva/clients/nmt/streaming_speech_translate_client.cc [hunk 2]
```diff
@@ -302,12 +304,15 @@ StreamingSpeechTranslateClient::ReceiveResponses(
         std::cout << "ASR started... press `Ctrl-C' to stop recording\n\n";
         gotoxy(0, 5);
       }
-      std::cout << "result: " << result.DebugString();
+      VLOG(1) << "result: " << result.DebugString();
       if (print_transcripts_) {
         call->AppendResult(result);
       }
+      std::cout << "translated text: \"" << result.alternatives(0).transcript() << "\"" << std::endl;
+      result_file << result.alternatives(0).transcript() << std::endl;
```
Comment (virajkarandikar) [inline riva/clients/nmt/streaming_speech_translate_client.cc:312]: should add a check here if result_file is valid (say if user sets nmt_text_file="")

Comment (rmittal-github) [reply riva/clients/nmt/streaming_speech_translate_client.cc:312]: I can add a check at the start if a valid value is provided

**Review by virajkarandikar — APPROVED** [review]

File: riva/clients/nmt/streaming_speech_translate_client.cc (hunk 2 above)
Comment (rmittal-github) [reply riva/clients/nmt/streaming_speech_translate_client.cc:312]: Discussed offline with Viraj, this should be ok for now. Resolving.

**Pull request merged**